tests/components/MeasurementExplorer.test.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable jest/no-standalone-expect */
+import { test, expect } from '@playwright/experimental-ct-react';
+
+import { MeasurementExplorer } from '../../src';
+import measurement from '../../stories/data/measurement.json';
+
+test.describe('MeasurementExplorer test', () => {
```

Comment:
Can you please remove all occurrences of "test" in the test names/descriptions? They are tests by definition.

Reply:
fixed, thank you